### PR TITLE
Fix(ui,utils): firstElementChild not in Edge (closes #798)

### LINKED ISSUE #798: firstElementChild is affecting MS Edge
@travisleithead reported: 

> Edge browser is broken when loading (live) respect documents for the past few days. Found (one of?) the root causes--lack of ElementTraversal support on the document node. After a quick scan of the respect code, it looks like only firstElementChild is used on the document

### PR DESCRIPTION
**Pull request by marcoscaceres**
This change is [Reviewable](https://reviewable.io/reviews/w3c/respec/799)

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@halindrome r?

**Comment by marcoscaceres** [comment]
@travisleithead, I don't have access to Edge, but I'm assuming this will work based on your previous PR.

**Comment by halindrome** [comment]
:lgtm:

<details><summary><i>Previously, marcoscaceres (Marcos Cáceres) wrote…</i></summary>
> @travisleithead, I don't have access to Edge, but I'm assuming this will work based on your previous PR. 
</details>


---

Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/respec/799#-:-KImPTEHGmVohGCsfgew:1417088721)*

File: js/core/utils.js [hunk 1]
```diff
@@ -34,13 +34,35 @@ define(
                 return function(doc, insertionPoint) {
                     node.remove();
                     doc.adoptNode(node);
-                    if (insertionPoint.firstElementChild) {
```
Comment (travisleithead) [inline js/core/utils.js]: If `insertionPoint` is never a `Document` object, then this is unnecessary, as all the ElementTraversal methods are supported on Element types in Edge :-)

**Comment by travisleithead** [comment]
Left a comment. It's only the document object itself that doesn't support the ElementTraversal APIs. So on Elements the extra complication is unnecessary.

**Comment by travisleithead** [comment]
Otherwise should be fine.

**Comment by marcoscaceres** [comment]
Ah, thanks for the clarification! I'll remove the wrapper and just use documentElement.

**Comment by marcoscaceres** [comment]
Merging this for now, as I'm going away for the weekend. Will remove the dead code next week, but at least it will be functional in Edge.

**Comment by travisleithead** [comment]
Thanks!

**Pull request merged**